packages/server/src/index.ts

```diff
@@ -1793,14 +1800,15 @@ export class App {
                 query.andWhere('chat_message.sessionId = :sessionId', { sessionId })
             }
 
-            // set date range
             query.andWhere('chat_message.createdDate BETWEEN :fromDate AND :toDate', {
-                fromDate: fromDate ?? new Date().setMonth(new Date().getMonth() - 1),
-                toDate: toDate ?? new Date()
+                fromDate: fromDate ?? new Date(new Date().setHours(0, 0, 0, 0)),
```

Comment:
I'd prefer to set the `fromDate` to a previous month

Reply:
Yes. Sure:
fromDate: fromDate ?? new Date(new Date(new Date().setMonth(-1)).setHours(0, 0, 0, 0)),